mobile/src/bundles/common/helpers/show-error-message/show-error-message.ts

```diff
@@ -0,0 +1,13 @@
+import { notifications } from '~/framework/notifications/notifications';
+
+import { ErrorMessages } from '../../enums/enums';
+
+const showErrorMessage = (error: unknown): void => {
+    if (error instanceof Error) {
+        notifications.showError(error.message);
+        throw error;
+    }
+    notifications.showError(ErrorMessages.UNKNOWN_ERROR);
+};
```

Comment:
I meant something like this
```suggestion
const getMessageFromError = (error: unknown): string => {
    if (error instanceof Error) {
       return error.message
    }
    return ErrorMessages.UNKNOWN_ERROR
};
```